plugins/cost.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: cost
+spec:
+  version: v0.1.4
+  shortDescription: View cluster cost information
+  description: |
+    Uses Kubecost's API to gather and format cluster cost information broken
```

Comment:
> Uses Kubecost's API

That is a bit vague, because most people won't know `kubecost` or its API. IIUC, kubecost is a cluster-side daemon that collects usage statistics. Maybe you can mention this here?

Reply:
Added a section to `description` explaining Kubecost in brief. What do you think?